Sources/Tracing/Span.swift

```diff
@@ -650,10 +650,26 @@ public struct SpanLink {
 
     /// Create a new `SpanLink`.
     /// - Parameters:
-    ///   - context: The `Baggage` identifying the targeted ``Span``.
+    ///   - baggage: The `Baggage` identifying the targeted ``Span``.
     ///   - attributes: ``SpanAttributes`` that further describe the link. Defaults to no attributes.
     public init(baggage: Baggage, attributes: SpanAttributes = [:]) {
         self.baggage = baggage
         self.attributes = attributes
     }
 }
+
+#if compiler(>=5.6)
+@preconcurrency public protocol _SwiftTracingSendableSpan: Sendable {}
+#else
+public protocol _SwiftTracingSendableSpan {}
+#endif
+
+#if compiler(>=5.6)
+extension SpanAttributes: Sendable {}
+extension SpanAttribute: @unchecked Sendable {}
```

Comment:
Maybe also explain why `@unchecked Sendable`?

Reply:
will do

Reply:
Agree with Moritz here, for each `@unchecked` we should have explanation why it can be unchecked.